_posts/2018-04-02-taldoor.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: TalDoor aka "GateKeeper V2"
+date: 2018-04-02
+categories:
+  - projects
+description: A small POE powered door lock system using the PocketBeagle along with a PN532 NFC module to scan and authenticate user ID cards and unlock the door.
+image: https://raw.githubusercontent.com/akaStanley/TalDoor/master/Board%20Photos/BothBoards_angle.PNG
+image-sm: https://raw.githubusercontent.com/akaStanley/TalDoor/master/Board%20Photos/BothBoards_angle.PNG
+author: Spencer Kulbacki
+author-image: https://avatars2.githubusercontent.com/u/14046811?s=460&v=4
+author-bio: 4th year Electrical Engineering Technology major and Research & Development Director 2017-18.
+author-email: sxk4788@rit.edu
+author-social:
+  github: https://github.com/akaStanley
+  linkedin: https://www.linkedin.com/in/skulbacki/
+---
+TalDoor is a hardware redesign of CSH's famous (now severely outdated and unsupported) iButton authentication system. TalDoor is named after our illustrious RA Tal, who was famous for having his door closed at all hours. TalDoor is a complete redesign of the original Gatekeeper system, with the main constraint of maintaining the same-footprint. 
+
+The two main complaints about the old GateKeeper system are that iButtons are insecure and finding custom power supplies for the hardware is difficult and costly. So when making TalDoor I chose to use the [PocketBeagle]( http://beagleboard.org/pocket) devboard. In addition to often returning pictures of cute dogs when googling the name, the PocketBeagle runs linux and is powerful enough to support encryption of the ID card data.
+
+To further the security of the system TalDoor uses the PN532 NFC reader which supports ISO14443A for use with encrypted Mifare DESFire NFC cards. You can read more about NFC and the PN532 [here]( https://learn.adafruit.com/adafruit-pn532-rfid-nfc/about-nfc). ID cards are fairly cheap and can be printed with names, photos, and even the CSH logo.
+
+Because the current GateKeeper system is run from custom 15v power supplies with inverted power/ground lines, finding replacement parts is difficult. Each GateKeeper system also had an ethernet connection for data. CSH was recently donated gigabit POE network switches that allow for up to 24 watts of power to be drawn from each ethernet port in the network. This means that TalDoor requires only a single ethernet cable for both power and data. At peak operation is uses a mere 3 watts of power which is well within the constraints.
+
+Additionally TalDoor has some extra goodies inside for enhanced operation. Here is a high level overview of the main components used in TalDoor:
+
+The main PCB which fits exactly where the old GateKeeper board lives in each door lock housing:
```

Comment:
This sentence reads a bit awkwardly. Maybe "The main PCB, which fits in the same space in the door lock housing as the old GateKeeper board, contains the following:"